spec

```diff
@@ -470,19 +469,19 @@ beginseg
     include "build/src/libultra_code/__osSpRawStartDma.o"
     include "build/src/libultra_code/__osSiRawStartDma.o"
     include "build/src/libultra_code/osSpTaskYield.o"
-    include "build/asm/code_801014C0.o"
-    include "build/data/code_801014C0.bss.o"
+    include "build/src/libultra_code/pfsreadwritefile.o"
+    include "build/data/contpfs.bss.o"
```

Comment:
If this is the bss for `contpfs.c` then it should be with it in the spec.

Reply:
oh i realized i accidentally had the wrong name on the file lol